tests/phpunit/tests/rest-api/rest-schema-setup.php

```diff
@@ -186,6 +189,10 @@ public function test_expected_routes_in_schema() {
 			'/wp-site-health/v1/tests/authorization-header',
 			'/wp-site-health/v1/tests/page-cache',
 			'/wp-site-health/v1/directory-sizes',
+			'/wp/v2/wp_font_family',
+			'/wp/v2/wp_font_family/(?P<id>[\d]+)',
+			'/wp/v2/wp_font_family/(?P<id>[\d]+)/autosaves',
+			'/wp/v2/wp_font_family/(?P<parent>[\d]+)/autosaves/(?P<id>[\d]+)',
```

Comment:
These now need to be updated to reflect the new routes

Reply:
Done!